client/post-editor/editor-ground-control/index.jsx

```diff
@@ -74,7 +78,9 @@ module.exports = React.createClass( {
 	},
 
 	setPostDate: function( date ) {
+		// TODO: REDUX - remove flux actions when whole post-editor is reduxified
 		postActions.edit( { date: date ? date.format() : null } );
+		this.props.setDate( date );
```

Comment:
It is moved to action, not the reducer, but yes, it's a conscious decision, and it's a good idea to put some information there!
I generally am trying to sneak logic out of the view, the view should not be concerned about the format that API takes data in